mesonbuild/cargo/interpreter.py

```diff
@@ -742,7 +746,10 @@ def load_wraps(source_dir: str, subproject_dir: str) -> T.List[PackageDefinition
     wraps: T.List[PackageDefinition] = []
     filename = os.path.join(source_dir, 'Cargo.lock')
     if os.path.exists(filename):
-        cargolock = T.cast('manifest.CargoLock', load_toml(filename))
+        try:
+            cargolock = T.cast('manifest.CargoLock', load_toml(filename))
+        except TomlImplementationMissing as e:
+            return wraps
```

Comment:
Maybe we should also print it as mlog.warning?

Reply:
I was unsure, mlog.log() or mlog.warning()? I see there is still the `as e` because I initially printed it... Maybe `mlog.warning(str(e), fatal=False)`?

Reply:
Went with non fatal warning.